File: Examples/DotNET/SDK/CSharp/ExtractTextFromImageUrl.cs
using System;
using Aspose.Cloud;

namespace Aspose.OCR.Cloud.Examples
{
    class ExtractTextFromImageUrl
    {
        static void Main()
        {
            string url = "http://cdn.aspose.com/tmp/ocr-sample.bmp";
            OCRLanguages language = OCRLanguages.English;
            bool useDefaultDictionaries = true;

            OCRResponse r = Common.OCRService.RecognizeImageTextFromUrl(
                url,
                language,
                useDefaultDictionaries
                );

            foreach (Part p in r.PartsInfo.Parts) {
                Console.WriteLine(p.Text);
            }

            Common.Pause();
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/ExtractTextFromImageUrl.vb
Imports System
Imports Aspose.Cloud

Namespace Aspose.OCR.Cloud.Examples
	Friend Class ExtractTextFromImageUrl
		Shared Sub Main()
			Dim url As String = "http://cdn.aspose.com/tmp/ocr-sample.bmp"
			Dim language As OCRLanguages = OCRLanguages.English
			Dim useDefaultDictionaries As Boolean = True

			Dim r As OCRResponse = Common.OCRService.RecognizeImageTextFromUrl(url, language, useDefaultDictionaries)

			For Each p As Part In r.PartsInfo.Parts
				Console.WriteLine(p.Text)
			Next p

			Common.Pause()
		End Sub
	End Class
End Namespace
